cmd/utils.go

```diff
@@ -59,3 +60,32 @@ func preRunConfig(cmd *cobra.Command, args []string) {
 		log.Info("config file not found, proceeding without it")
 	}
 }
+
+func createArtifactsDir(artifactsDir string) (string, error) {
```

Comment:
IMO, the artifact directory creation should be in preRunConfig() method

Reply:
Let me check.

Reply:
I do see the preRunConfig() includes configuration and file creation. So I 
I can move both the dir and file creation of artifacts/results.json to preRunConfig() if that makes more sense.

Reply:
In my opinion yes.. It would be great to have Brad's insight as well :) @bcrochet

Reply:
It looked like a mess to move the file creation and less of a mess to move the dir creation so I did the latter.

Reply:
I made a comment above about where I believe it should occur.